### PR TITLE
[HOPSWORKS-35] Add spark prefix on Spark metrics

### PR DESCRIPTION
**Pull request by Fil0x**
Metrics written to graphite from Spark are prefixed with spark.*. This allows for templates in influxdb to pick up and transform Spark metrics.

### CODE REVIEW

**Comment by Fil0x** [comment]
This change breaks grafana display, working on a pull request on hopsmonitor to work with the prefix.

File: Berksfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ source 'https://supermarket.chef.io'
 
 cookbook 'java'
 cookbook 'kagent', github: "karamelchef/kagent-chef", branch: "master"
-cookbook 'hops', github: "hopshadoop/hops-hadoop-chef", branch: "master"
```
Comment (jimdowling) [inline Berksfile]: You need to remove these changes in the Berksfile.

File: Vagrantfile [hunk 2]
```diff
@@ -11,52 +11,52 @@ Vagrant.configure("2") do |c|
     c.cache.enable :apt
     c.cache.enable :gem    
   end
-  c.vm.box = "bento/ubuntu-16.04"
-#  c.vm.box = "opscode-ubuntu-14.04"
```
Comment (jimdowling) [inline Vagrantfile]: Again, remove these. We have moved to ubuntu 16.04 as the default platform.

File: Vagrantfile [hunk 3]
```diff
@@ -11,52 +11,52 @@ Vagrant.configure("2") do |c|
...
-#  c.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-14.04_chef-provisionerless.box"
+#  c.vm.box = "bento/ubuntu-16.04"
+  c.vm.box = "opscode-ubuntu-14.04"
+  c.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-14.04_chef-provisionerless.box"
 #   c.vm.box = "bento/centos-7.2"
 #  c.vm.box_url = "https://atlas.hashicorp.com/ubuntu/boxes/trusty64/versions/20150924.0.0/providers/virtualbox.box"
 #  c.vm.hostname = "default-ubuntu-1404.vagrantup.com"
```
Comment (jimdowling) [inline Vagrantfile]: Same here.

File: templates/default/metrics.properties.erb [hunk 4]
```diff
@@ -121,7 +121,7 @@
 *.sink.graphite.host=<%= @private_ip %>
 *.sink.graphite.port=<%= node.influxdb.graphite.port %>
 *.sink.graphite.period=1
-
+*.sink.graphite.prefix=spark
```
Comment (jimdowling) [inline templates/default/metrics.properties.erb]: This 1 line should be the PR

**Pull request closed without merging**